Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM ubuntu:18.04
+
+# Set up the non-root user
+RUN apt-get update \
+    &&  apt-get -y install sudo \
+    && useradd -ms /bin/bash user && echo "user:user" | chpasswd && adduser user sudo
+
+ADD /sudoers.txt /etc/sudoers
+
+ENV ECLIPSER_HOME /home/user/Eclipser
+
+WORKDIR /home/user
+
+COPY . /home/user/deepstate
```

Comment:
Because of this statement that gets the source code into the `deepstate` directory, it is owned by `root`. In order to make changes later, we need to `chown` to `user`